rusk/src/lib/chain/rusk.rs

```diff
@@ -77,14 +78,19 @@ impl Rusk {
 
     pub fn execute_transactions<I: Iterator<Item = Transaction>>(
         &self,
-        block_height: u64,
```

Comment:
By limiting EST call elapsed time we make sure that the block generator will most probably propagate the Candidate to all provisioners before their Proposal step times out.

However, don't we have a similar problem with VST call too? If a low-spec committee member waits for VST to execute for more than `Validation timeout` , it may propagate its vote too late, thus being discarded.

Reply:
We cannot limit the VST, the validator need to fully verify the block

Reply:
Or do you mean to limit EST in a way that VST will not fail?

In that case, what about getting the MIN of this two steps instead of getting only the proposal one?

Reply:
Makes sense or we can simply always run `Validation step` with N % higher timeout than Proposal. 

TBH not sure yet, maybe it's worth tackling it with another PR and after collecting more ITN/devnet data. Ideally, in devnet we should expect to see a `full block` always produced at iteration 0.